File: C/1008.c
#include <stdio.h>

int main ()
{
  int A, B;
  double C, salary;
  scanf ("%d %d %lf", &A, &B, &C);
  salary = B * C;
  printf("NUMBER = %d\n", A);
  printf ("SALARY = U$ %.2lf\n", salary);
  return 0;
}

File: C#/1008.cs
using System;
					
public class Program
{
	public static void Main()
	{
		int A, B;
		double C, salary;
		A = int.Parse(Console.ReadLine());
		B = int.Parse(Console.ReadLine());
		C = double.Parse(Console.ReadLine());
		salary = B * C;
		Console.WriteLine("NUMBER = " + A);
		Console.WriteLine("SALARY = U$ " + salary.ToString("F2"));
	}
}